### PR TITLE
Bryanv/4889 changelog

### LINKED ISSUE #4889: Simplify CHANGELOG
Currently the auto-generated `CHANGELOG` attempts to copy parts of the release notes over. This is both fragile (a migration guide section will confuse the output greatly) as well as duplicative. Proposed to clean up the existing `CHANGELOG` to only have issues, and modify the script to not perform the copying in the future.

### PR DESCRIPTION
**Pull request by bryevdv**
issues: closes #4889

@damianavila @bokeh/dev propose to merge this last PR for `0.12.1`

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -770,149 +734,6 @@
...
-    - #3751 [bokehjs] Remove unused close_wrapper module and coffee/api directory
-    - #3752 [bokehjs] [build] Generate defaults for test task, not scripts:coffee
-    - #3755 [docs] Quickstart edits
-    - #3766 [bokehjs] Task/rename hasproperties to hasprops
-    - #3775 Removing blazedatasource
-    - #3790 Don't use spaces in json separators to reduce data size
-    - #3802 [bokehjs] Move vbox and hbox to top since they are layouts on coffee layer
-    - #3805 [build] Add new creds for rackspace
-    - #3808 [docs] Fix quotes around links to examples notebooks
-    - #3815 Hotfix/apps examples
```
Comment (bryevdv) [inline CHANGELOG]: The `0.11.1` section was entirely duplicated

**Comment by bryevdv** [comment]
Have tested locally, performs as desired.

**Comment by fpliger** [comment]
LGTM. Will merge this within 1 hr if there are no objections :)

**Comment by damianavila** [comment]
LGTM, simplify things...

**Pull request merged**